skiko/src/jvmMain/kotlin/org/jetbrains/skiko/PlatformOperations.kt

```diff
@@ -11,6 +10,8 @@ import java.awt.Component
 import java.awt.Window
 import javax.swing.SwingUtilities
 
+internal var testNonSoftwareRedrawer: ((SkiaLayer) -> Redrawer)? = null
```

Comment:
This kind of top level variable looks somewhat ugly. Why cannot we keep this kind of state out of the top level?

Reply:
Moved it into factory